scripts/create-pr-with-vrt.sh

```
@@ -0,0 +1,402 @@
+
+    log_info "VRT 結果を解析中..."
+    
+    # jqでJSONを解析
+    if ! command -v jq &> /dev/null; then
+        log_warning "jq が見つかりません。jq なしでVRT結果を処理します。"
+        generate_vrt_fallback_content
+        return 0
+    fi
+
+    # VRT結果を解析
+    local success=$(jq -r '.success' "$VRT_RESULTS_FILE" 2>/dev/null || echo "false")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)